screens/backupAndRestore/BackupAndRestoreScreen.tsx

```diff
@@ -199,17 +220,18 @@ const BackupAndRestoreScreen: React.FC<BackupAndRestoreProps> = props => {
       arrowLeft={true}
       headerRight={
         <HelpScreen
-          source={'BackUp'}
+          source={'Inji'}
           triggerComponent={
-            <Icon
-              {...testIDProps('help')}
-              accessible={true}
-              name="question"
-              type="font-awesome"
-              size={21}
-              style={Theme.Styles.IconContainer}
-              color={Theme.Colors.Icon}
-            />
+            <View testID="helop" style={Theme.HelpScreenStyle.viewStyle}>
```

Comment:
helop -> should this be help?
Also if you are renaming the view's testID to help, please rename Text (L: 230) testID to helpText. Because I have noticed that if we have 2 testIDs with same name in a screen, it wont be visible in appium inspector

Reply:
Changed